package.json

```diff
@@ -25,7 +25,9 @@
     "test": "yarn workspace client run test"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^3.3.0"
+    "@openzeppelin/contracts": "^3.3.0",
+    "clipboard-react": "^1.0.0",
+    "react-share": "^4.3.1"
```

Comment:
I think these were meant to be added to the client package.json only. Can you uninstall these from the root package, `yarn remove clipboard-react react-share`.